pkg/cmd/extension/manager.go

```diff
@@ -64,6 +65,14 @@ func (m *Manager) SetClient(client *http.Client) {
 	m.client = client
 }
 
+func (m *Manager) EnableDryRunMode() {
+	m.dryRunMode = true
+}
+
+func (m *Manager) DisableDryRunMode() {
```

Comment:
Is Disable called from anywhere?

Reply:
It is not right now, I added it mostly because it felt weird having a `Enable` function without the corresponding `Disable` function. Having said that, I don't know if we will ever need the ability to disable it so I will remove it for now. It is easy enough to add later.